src/contact.rs

```diff
@@ -83,6 +83,9 @@ pub struct Contact {
 pub enum Origin {
     Unknown = 0,
 
+    /// The contact is a mailing list address, needed to unblock mailing lists
+    MailinglistAddress = 0x2,
```

Comment:
This update logically belongs to the last commit, not second. Just something I noticed reviewing each commit separately.

Reply:
i moved it down, was on the last commit anyway :)